docs/architecture/deployment/private-saas/download-logs.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Download Logs
+id: download-logs
+description: Download Prophecy support logs
+sidebar_position: 3
+tags:
+  - download
+  - support logs
+---
+
+As an admin user, you can download your environment logs from Prophecy without needing access to your Prophecy cluster or the assistance of Prophecy Support. This reduces the delay in debugging any issues with your Prophecy services.
+
+## Use case
+
+- The overarching objective is to debug what is going on with your Prophecy services.
+- To achieve this, we've enabled admins to be able to download Prophecy logs and environment information so that they can upload them to Zendesk.
+
+### Download Logs features
+
+You can use the Download Logs feature to capture logs using the Services and time selectors.
+
+- All services are selected by default.
+- The download supports one hour of logs from the Start Time.
+
+The captured logs include all relevant Kubernetes services, including sandbox pods.
+
+- Configuration Data Collection:
+
+  - Captures Kubernetes cluster configuration, such as resource quotas and node configuration.
```

Comment:
we capture prophecy configuration i.e. prophecy CR and config map.

Reply:
Done.